python-checks/src/test/resources/checks/methodShouldBeStatic.py

```diff
@@ -133,6 +133,37 @@ def fun31(slf): raise ValueError('Some error') # Noncompliant
     def fun25(slf, other):
         # some comment
         raise NotImplementedError
-                
+
+    # from here on, we test built-in functions
+
+    def __init__(slf):
+        print(1)
+
+    def __init__(slf):
+        print(slf)
+
+    def __this_is_not_a_built_in_method__(slf):
```

Comment:
@vilchik-elena methods such as __init__ are really built in. Defining them in a class has a particular meaning (overriding). Method __this_is_not_a_built_in_method__ is not a built-in method, although for simplicity our implementation considers it as such